integreat_cms/cms/models/abstract_content_translation.py

```diff
@@ -436,25 +436,26 @@ def translation_state(self):
             # If the page does not have a major public version, it is considered "missing" (keep in mind that it might
             # have draft versions or public versions that are marked as "minor edit")
             return translation_status.MISSING
-        if translation.machine_translated:
-            return translation_status.MACHINE_TRANSLATED
         if translation.currently_in_translation:
             return translation_status.IN_TRANSLATION
         if not self.source_language:
             # If the language of this translation is the root of this region's language tree, it is always "up to date"
             return translation_status.UP_TO_DATE
-        if not (source_translation := self.major_source_translation):
+        if (
             # If the source language does not have a major public version, the translation is considered "outdated",
             # because the content is not in sync with its source translation
-            return translation_status.OUTDATED
-        if source_translation.translation_state == translation_status.OUTDATED:
+            not (source_translation := self.major_source_translation)
             # If the source translation is already outdated, this translation is as well
+            or source_translation.translation_state == translation_status.OUTDATED
+            # If the translation was edited before the last major change in the source language, it is outdated
+            or translation.last_updated < source_translation.last_updated
```

Comment:
```suggestion
            or translation.last_updated <= source_translation.last_updated
```
Haha, I thought the case that two translations are created in exactly the same second is a rare enough edge case that we can ignore it - apparently, in the test data there are translations with exactly the same date and they expect the translations to be flagged as outdated in this case...
I'm fine with either way - adapting the tests or make this if-condition compliant with the tests^^